Python/Tests/Core/PathUtilsTests.cs

```diff
@@ -31,7 +31,7 @@ public class PathUtilsTests {
             AssertListener.Initialize();
         }
 
-        [TestMethod, Priority(0)]
+        [TestMethod, Priority(TestExtensions.IMPORTANT_UNIT_TEST)]
```

Comment:
i'd like to keep all these pathutilstests

Reply:
Changed